deployments/optimism/.chainId

```diff
@@ -0,0 +1 @@
+10
```

Comment:
what's this file for?

Reply:
It's the chain ID for the network, see: https://chainlist.org/